client/src/components/LoginSession/LoginSessionContainer.js

```diff
@@ -34,27 +34,32 @@ function LoginSessionContainer(props) {
       id: props.ID,
       SecurityID: Config.get('SecurityID')
     })
-      .then(response => {
-        const failure = !response.success;
-        setFailed(failure);
-
-        if (failure) {
-          props.displayToastFailure(response.message);
-        } else {
-          props.displayToastSuccess(response.message);
+    .then(response => {
```

Comment:
I've reorganised this so we always use HTTP error code. The main `then` statement will only be called when success status code is attached to our response